pdart/astroquery/Astroquery.py

```diff
@@ -138,19 +108,28 @@ def get_proposal_ids(self) -> List[int]:
             self.proposal_ids = sorted(list(set(result)))
         return self.proposal_ids
 
-    def get_products(self, proposal_id: int) -> Table:
+    def get_products(
+        self, proposal_id: int, selected_suffixes: Optional[bool] = False
```

Comment:
Using `bool` instead of `Optional[bool]` would be clearer.  If the parameter is missing in a call, the initialization will use `False` anyway.